cmd/process-agent/app/check.go

```diff
@@ -174,7 +182,21 @@ func printResultsBanner(name string) {
 	fmt.Printf("-----------------------------\n\n")
 }
 
-func printResults(msgs []process.MessageBody) error {
+func printResults(check string, msgs []process.MessageBody) error {
+	if checkOutputJSON {
+		return printResultsJSON(msgs)
+	}
+
+	err := checks.HumanFormat(check, msgs, os.Stdout)
+	switch err {
+	case checks.ErrNoHumanFormat:
+		return printResultsJSON(msgs)
+	default:
+		return err
+	}
```

Comment:
2 things: 
1. Do we want to print the error?
2. For the default case do can we also not print the json version?

Reply:
Yes, I am still debating this. I think not printing anything by default make break assumptions for other check owners.

Reply:
I think we should probably display some sort of message, at least to tell the user that we are falling back to the json output. Otherwise they may be a bit confused when they unexpectedly get json output with some checks.

Reply:
I think that this is likely to happen with the `connections` check. Like Corrina said, we can probably log a message saying that human readable output is not yet supported and we're falling back to the json outpout.

Reply:
I like the idea of showing the message.

Reply:
I think cleaning up the JSON is something that we need to address separately. You cannot pipe it to `jq` at the moment.

Reply:
I didn't realize that we couldn't actually pipe the output to `jq`. The fix for that would be to pipe all of the process agent logs to `stderr`. I'd be happy to work on that separately.

Reply:
Not all of these logs are errors so the info logs should just be suppressed. But good call on splitting this and implementing separately.